recipes/qt-opcua/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set version = "6.6.2" %}
+
+package:
+  name: qt6-opcua
+  version: {{ version }}
+
+source:
+  url: https://github.com/qt/qtopcua/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 3c7210493dbdf6c7ddd3fa99fd8b53510eb903cbcd2f95571d65b20eb2bc9cb2
+  patches:
+    - fix_nonamalgamated_open62541.patch
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  run_exports:
+    - {{ pin_subpackage('qt6-opcua', max_pin='x.x') }}
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - perl
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - qt6-main {{ version }}             # [build_platform != target_platform]
+    - sysroot_linux-64 2.17              # [linux]
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - {{ cdt('mesa-libegl-devel') }}     # [linux]
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
+    - {{ cdt('libdrm-devel') }}          # [linux]
+    - {{ cdt('libglvnd-glx') }}          # [linux]
+    - {{ cdt('libglvnd-egl') }}          # [linux]
+  host:
+    - open62541
+    - openssl
+    - qt6-main {{ version }}
+
+test:
+  requires:
+    - sysroot_linux-64 2.17              # [linux]
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - {{ cdt('mesa-libegl-devel') }}     # [linux]
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
+    - {{ cdt('libdrm-devel') }}          # [linux]
+    - {{ cdt('libglvnd-glx') }}          # [linux]
+    - {{ cdt('libglvnd-egl') }}          # [linux]
+    - cmake
+    - ninja
+  source_files:
+    - examples/
+  commands:
+    {% set qt_libs = ["OpcUa"] %}
+    {% for each_qt_lib in qt_libs %}
+    - test -d $PREFIX/include/qt6/Qt{{ each_qt_lib }}  # [unix]
+    - test -f $PREFIX/lib/libQt6{{ each_qt_lib }}${SHLIB_EXT}  # [unix]
+    - if not exist %PREFIX%\\Library\\include\\qt6\\Qt{{ each_qt_lib }} exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\lib\\Qt6{{ each_qt_lib }}.lib exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\bin\\Qt6{{ each_qt_lib }}.dll exit 1  # [win]
+    {% endfor %}
+
+about:
+  home: http://qt-project.org
+  license: LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+  license_file:
+    - LICENSES/LGPL-3.0-only.txt

```

Comment:
the doc only mentions GPL3, not LGPL:
https://doc.qt.io/qt-6/qtcoap-index.html

Reply:
> the doc only mentions GPL3, not LGPL: https://doc.qt.io/qt-6/qtcoap-index.html

That page seems to refer to qt-coap, not qt-opcua, in the qp-opcua they refer to both LGPL and GPL : https://doc.qt.io/qt-6/qtopcua-overview.html#licenses . For the string here, I took what they used in the source code: https://github.com/qt/qtopcua/blob/v6.6.2/src/opcua/core/qopcuaplugin.h#L2 . There is still a slight inconsistencies between doc and source code as in they doc they do not explicit refer to GPL-3.0, but I guess because the docs are less detailed w.r.t. to the source code as GPL-3.0 licensing is kind of implied by LGPL-3.0 . I wanted to report this, but I could not without a Qt account, so I gave up. If you prefer, I can remove any reference to GPL-3.0 here.

Reply:
oh yes wrong page, sorry